spec/features/requester_edits_ncr_work_order_spec.rb

```diff
@@ -7,10 +7,11 @@
 
   let(:work_order) { create(:ncr_work_order, description: 'test') }
   let(:ncr_proposal) { work_order.proposal }
-  let!(:approver) { create(:user) }
+  let!(:approver) { create(:user, client_slug: 'ncr') }
```

Comment:
Actually, that behavior is why I don't think we want it to be a constant. A typo on the string *would* throw a validation exception because `User.client_slug` validates against `Proposal.client_slugs`.

I'm -0 on the change. I think keeping it a string keeps exercising the validation and doesn't mean needing to type-convert anywhere.